pkg/config/templates/patch/rancher-logging/100.1.3+up3.17.7/eventtailer.yaml

```diff
@@ -0,0 +1,15 @@
+{{- with $.Values.eventTailer }}
```

Comment:
should we drop the artifacts for 100.1.3+up3.17.7 since the version-logging is pointing to `LOGGING_VERSION=102.0.0+up3.17.10`

Reply:
Good point. This is a general question/release policy.

For instance, in the upgrade script, we have many codes with condition of specific release/version.

harvester/harvester/package/upgrade:
![image](https://user-images.githubusercontent.com/31133476/231699713-1d975d41-94ab-47c9-b430-0ae2c8c8ffc1.png)


At the moment, we keep all of them and not deleting the past code/artifacts.

cc @bk201